### PR TITLE
User show: Use badge for showing the role

### PR DESCRIPTION
**Pull request by ChrisBr**
selection_006

Before 

selection_005

### CODE REVIEW

File: src/api/app/views/webui2/webui/user/_info.html.haml [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
-    %h4.home-usertitle.mt-4.mb-3
+    %h4.home-usertitle.mt-4
       #home-realname
         = user.realname
-      .mt-2#home-login
-        = user.login
+    %h4.d-inline-block#home-login
+      = user.login
+    - role_titles.each do |title|
+      %span.badge.badge-secondary.align-top
```
Comment (dmarcoux) [inline src/api/app/views/webui2/webui/user/_info.html.haml]: I would add `mt-1` here to align this with the user login

File: src/api/app/views/webui2/webui/user/_info.html.haml [hunk 2]
```diff
@@ -21,15 +24,9 @@
           %li
             = link_to(group_show_path(group)) do
               #{group.title}
-              %span.badge.badge-primary
+              %span.badge.badge-primary.align-top
```
Comment (dmarcoux) [inline src/api/app/views/webui2/webui/user/_info.html.haml]: Same here, `mt-1` to align it with the group title

**Comment by coolo** [comment]
So you're saying you became root? Cool hacker!

**Comment by vpereira** [comment]
nice @ChrisBr , but please squash both commits in one

File: src/api/app/views/webui2/webui/user/_info.html.haml (hunk 1 above)
Comment (ChrisBr) [reply src/api/app/views/webui2/webui/user/_info.html.haml]: Actually I like it a lot more when it is aligned to the top. It is additional information to the user login. 

selection_008

vs

selection_007

**Comment by ChrisBr** [comment]
>  but please squash both commits in one

Why? The two commits are unrelated!

File: src/api/lib/backend/remember_location.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+module Backend
+  module RememberLocation
+    def singleton_method_added(method_name)
```
Comment (bgeuken) [inline src/api/lib/backend/remember_location.rb]: @ChrisBr This slipped in by accident, right?

Comment (ChrisBr) [reply src/api/lib/backend/remember_location.rb]: Oh yeah, removing

**Pull request merged**